Server/Modules/MapGenerator/Services/MapGeneratorService.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Fracture.Server.Modules.MapGenerator.Models;
+using Fracture.Server.Modules.NoiseGenerator.Models;
+using Fracture.Server.Modules.NoiseGenerator.Services;
+using System.Numerics;
+
+namespace Fracture.Server.Modules.MapGenerator.Services;
+
+public class MapGeneratorService : IMapGeneratorService
+{
+    private MapData _mapData;
+
+    private float persistance = 0.5f;
+    private float lacunarity = 2f;
+    private int octaves = 5;
+    private float scale = 5f;
+
+    private Random rnd = new Random();
+
+    //private Dictionary<(TemperatureType, HeightType), Biome> biomesDictionary = new Dictionary<(TemperatureType, HeightType), Biome>
```

Comment:
unless this code is to be used later, i don't think we should keep commented out code